shared/middleware/animation_library_api.rb

```diff
@@ -110,6 +110,38 @@ class AnimationLibraryApi < Sinatra::Base
     not_found
   end
 
+  #
+  # GET /api/v1/animation-library/default_spritelab/
+  #
+  # Retrieve the default sprite list from S3
+  get %r{/api/v1/animation-library/default_spritelab} do
```

Comment:
i'm not sure what our convention is on the backend, but we should be consistent with `-` and `_` in URLs. it looks like i missed this when we added `/api/v1/animation-library/level_animations/` -- i think it's fine to revisit and rename these routes, but we should log that as a task so we don't forget

Reply:
Related PR here: https://github.com/code-dot-org/code-dot-org/pull/41679